pkg/controller/master/backup/restore.go

```diff
@@ -748,9 +755,24 @@ func (h *RestoreHandler) deleteOldPVC(vmRestore *harvesterv1.VirtualMachineResto
 }
 
 func (h *RestoreHandler) startVM(vm *kubevirtv1.VirtualMachine) error {
-	logrus.Infof("starting the vm %s, current state running:%v", vm.Name, *vm.Spec.Running)
-	if vm.Spec.Running == nil || !*vm.Spec.Running {
+	runStrategy, _ := vm.RunStrategy()
+	logrus.Infof("starting the vm %s, current state running:%v", vm.Name, runStrategy)
+
+	switch runStrategy {
+	case kubevirtv1.RunStrategyAlways:
+		return nil
```

Comment:
Could you explain why there is no need to start VM if the strategy is `RunStrategyAlways`?

Reply:
`RunStrategyAlways` will start VM automatically
same as `running: true`
and `RunStrategyAlways` is not possible to stop, kubevirt will start the stopped VM automatically until we setup VM as RunStrategyHalted